tests/test_native_runtime_assets.py

```
@@ -0,0 +1,378 @@
+    )
+
+
+def test_required_docs_are_not_gitignored() -> None:
+    docs = [
+        "docs/native-linux-deploy-guide.md",
+        "docs/docker-free-fastapi-deploy-runbook.md",
+        "legacy/docker-deploy/README.md",
+        "legacy/bootstrap/README.md",
+    ]
+
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)